staging/src/k8s.io/apiextensions-apiserver/test/integration/validation_test.go

```diff
@@ -47,8 +47,8 @@ func TestForProperValidationErrors(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	te, ns := "not-the-default-te", "not-the-default-ns"
-	noxuResourceClient := newNamespacedCustomResourceClient(te, ns, dynamicClient, noxuDefinition)
+	ns := "not-the-default-ns"
+	noxuResourceClient := newNamespacedCustomResourceClient(ns, dynamicClient, noxuDefinition)
```

Comment:
Remove tenant, is this right?

Reply:
The code was never in CI/CD before. The tenant part must have been accidentally change in the past. We need to remove it to make the code build.

The multi-tenancy change will be done in a different PR.